django_sharding_library/settings_helpers.py

```diff
@@ -89,4 +90,13 @@ def database_configs(databases_dict):
                 )
                 if db_config:
                     configuration[replica['name']] = db_config
+
+            # We assume the numeric shard ID is constant based on the entries in the configuration helper (we assume
+            # they wont change order, and that new shards will be appended and not inserted randomly)
+            # todo: update docs to make sure this assumption is understood
```

Comment:
Awesome, a bit out of commission for a few days but later this week I'll
get the Travis pull request merged as well as take another look at your code

On Sun, May 15, 2016, 7:50 PM Titus Peterson <notifications@github.com>
wrote:

> In django_sharding_library/settings_helpers.py
> <https://github.com/JBKahn/django-sharding/pull/26#discussion_r63300509>:
>
> > @@ -89,4 +90,13 @@ def database_configs(databases_dict):
> >                  )
> >                  if db_config:
> >                      configuration[replica['name']] = db_config
> > +
> > +            # We assume the numeric shard ID is constant based on the entries in the configuration helper (we assume
> > +            # they wont change order, and that new shards will be appended and not inserted randomly)
> > +            # todo: update docs to make sure this assumption is understood
>
> Doc updated with this assumption
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/JBKahn/django-sharding/pull/26/files/4e531964efbf77ff8c4fb4d5a57829a49a1b521c#r63300509>
>